packages/tests/integration/src/entities/Task.ts

```diff
@@ -2,7 +2,8 @@ import { TaskCodegen } from "./entities";
 
 import { taskConfig as config } from "./entities";
 
-export class Task extends TaskCodegen {}
```

Comment:
@stephenh - I don't know how important testing a concrete base class was. I needed to make it abstract for my test cases, and there wasn't too much changes needed elsewhere to make the desired assertions.

Let me know if you'd like me to introduce a new STI table to keep this one concrete

Reply:
Np on making `Task` abstract; I had been doing the same when debugging locally :-) :shrug: